docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,73 @@
+# Migrate from another CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea.
+
+NOTE: The following is a reference list of CNIs and versions we have already
+verified the migration process. CNIs and versions that are not listed here
+might also work. Please create an issue if you run into problems during the
+migration to Antrea. During the migration process, no Kubernetes resources
+should be created or deleted, otherwise the migration process might fail or
+some unexpected problems might occur.
+
+| CNI     | Version |
+|---------|---------|
+| Calico  | v3.26   |
+| Flannel | v0.22.0 |
+
+The migration process is divided into three steps:
+
+1. Clean up the old CNI.
+2. Install Antrea in the cluster.
+3. Restart all Pods in the cluster in-place.
+
+## Clean up the old CNI
+
+Before installing Antrea in the cluster, the old CNI needs to be cleaned up. The
+cleanup process is different for different CNIs, but the basic idea is to
+remove the old CNI DaemonSet, Deployment, and CRDs from the cluster. The most
+common way is to use the `kubectl delete` command to delete the old CNI, just
+like the following example:
```

Comment:
The cleanup process varies across CNIs, typically you should remove the DaemonSet, Deployment, and CRDs of the old CNI from the cluster. For example, if you used `kubectl apply -f <CNI_MANIFEST>` to install the old CNI, you could then use `kubectl delete -f <CNI_MANIFEST>` to uninstall it.

And the example below can be removed.

Reply:
Addressed.